infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java

```diff
@@ -190,13 +190,12 @@ public ConflictTxStreamEntry(ConflictTxStream stream, long address) {
     }
 
     class ConflictTxStreamEntryComparator implements Comparator<ConflictTxStreamEntry> {
-        // Overriding compare()method of Comparator
-        // for descending order of cgpa
+        // Overriding compare()method of Comparator for ascending order
         public int compare(ConflictTxStreamEntry s1, ConflictTxStreamEntry s2) {
```

Comment:
can be made concise by returning (s1.txVersion-s2.txVersion)

Reply:
Long.compare(s1.txVersion, s2.txVersion)